.gitignore

```diff
@@ -6,6 +6,11 @@ github_search*.jsonl
 schemastore
 .venv*
 scratch.py
+perf.data
+perf.data.old
+flamegraph.svg
+# Additional target directories that don't invalidate the main compile cache when changing linker settings
+/target*
```

Comment:
For the future: I don't love adding entries to `.gitignore` that are effectively "personal" because it becomes hard to keep track of why they were added, when they're assumed to be present, etc. I personally :) would add these to my global gitignore.